src/edgekv/ekv-helper.ts

```diff
@@ -356,6 +356,10 @@ export function getDateDifference(date) {
 
 export function convertDaysToSeconds(days: number) {
     if (!isNaN(days) && days >= 0) {
+        if (days != 0 && (days < 1 || days > 3650)){
```

Comment:
I think both these conditions (days != 0 && days < 1) are already covered in "days >= 0". Would recommend moving the max check to the outer loop.
if (!isNaN(days) && (days >= 0  || days > 3650))

Reply:
@swathimr Any value between 0 and 1 is invalid (eg 0.5), but the value 0 is still a valid retention period (unlimited). Not sure if your suggestion would still handle that case correctly?

I was trying to make the if statement clear by saying that the range is only required if `days` is not zero. Perhaps I could add a code comment to clarify this?

Reply:
@swathimr 
We get the same error message from the create namespace edgeKV API for very large value of retention period. Thanks @slukic0 for finding it. 
At the CLI level, we are just outputting the same error message in console that we get from the create namespace edgeKV API. So, if we add this limit condition at the CLI level and later limit for retention period gets changed at the API level, we would need to change it at the CLI level again. Does it make sense to just fix it at the API level or is it fine to add limit conditions in CLI codebase also?

Reply:
talked offline and sorted it out.